docs/client/serving/kn-create-service.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "kn service create"
+weight: 04
+type: "docs"
+---
+<!-- TODO: make a reusable snippet-->
```

Comment:
plz keep adding this to things, will be helpful later

Reply:
bonus points if you can show where the snippet would end with something like `<!---->`